apps/src/constants.js

```diff
@@ -65,11 +67,17 @@ exports.TestResults = {
 
   // The level was solved in an optimal way.
   FREE_PLAY: 30,   // The user is in free-play mode.
+  MINIMUM_OPTIMAL_RESULT: 30,
```

Comment:
I wonder if we should should have more detailed comments about this whole numbering system.  For example, not sure what it means to be minimal + optimal at the same time :)